go/enclave/components/batch_registry.go

```diff
@@ -59,6 +60,8 @@ func (br *batchRegistryImpl) UnsubscribeFromEvents() {
 // StoreBatch - stores a batch and if it is the new l2 head, then registry will update
 // stored head pointers
 func (br *batchRegistryImpl) StoreBatch(batch *core.Batch, receipts types.Receipts) error {
+	defer br.logger.Info("Registry StoreBatch() exit", "batch", batch.Hash(), "duration", measure.NewStopwatch())
```

Comment:
there is a log.BatchHashKey instead of "batch"

you could create one for "duration" as well